ask-sdk-core/src/com/amazon/ask/CustomSkill.java

```diff
@@ -119,7 +127,8 @@ protected ResponseEnvelope invoke(final UnmarshalledRequest<RequestEnvelope> unm
         JsonNode requestEnvelopeJson = unmarshalledRequest.getRequestJson();
 
         if (skillId != null && !requestEnvelope.getContext().getSystem().getApplication().getApplicationId().equals(skillId)) {
-            throw new AskSdkException("AlexaSkill ID verification failed.");
+            LOGGER.debug("AlexaSkill ID verification failed.");
```

Comment:
nit.. add the expected and actual skill id to the log message here

Reply:
Will update with a new commit